src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -19,7 +19,7 @@ public class AutomationIntegrationTests
         // These values should change only in response to intentional rule modifications
         const int WildlifeManagerKnownErrorCount = 12;
         const int Win32ControlSamplerKnownErrorCount = 0;
-        const int WindowsFormsControlSamplerKnownErrorCount = 6;
+        const int WindowsFormsControlSamplerKnownErrorCount = 4;
```

Comment:
Sampler app has both an edit control and a spinner control, each of which used to fail.